index.html

```diff
@@ -144,15 +154,62 @@ <h2 class="title">Uber Navigation</h2>
                     <li class="other-skills">css</li>
                     <li class="other-skills">javascript</li>
                 </ul>
-                <button class="card-btn">See Project</button>
+                <button id="click" class="card-btn" onclick="openPopup()">See Project</button>
```

Comment:
- [ ] Just as the previous reviewer suggested, You need to remove the unused code comment from your code. It makes your code base look unprofessional. :+1: :100: